j5/boards/base.py

```diff
@@ -23,6 +25,14 @@ def __repr__(self) -> str:
         """A representation of this board."""
         return f"<{self.__class__.__name__} serial={self.serial}>"
 
+    def setup(self):
```

Comment:
Requires caller to remember to call this. Is there a way to make this mandatory? (May not be just a thought)

Reply:
Not easily, documented in #28 for future fixing as discussed.